gufe/ligandnetwork.py

```diff
@@ -211,7 +211,10 @@ def remove_edges(self, edges: Union[LigandAtomMapping, list[LigandAtomMapping]])
                              f"{extras}")
 
         new_edges = current - to_remove
-
+new_net = LigandNetwork(new_edges, self.nodes)
+if not new_net.graph().is_connected():
```

Comment:
```suggestion
if not self.is_connected():
```

sorry just saw, there is the functionality inside the class

Reply:
Absolutely correct!